plugins/serializers/splunkmetric/splunkmetric.go

```diff
@@ -68,9 +68,18 @@ func (s *serializer) createObject(metric telegraf.Metric) (metricGroup []byte, e
 
 	for _, field := range metric.FieldList() {
 
-		if !verifyValue(field.Value) {
+		switch field.Value.(type) {
+		case string:
 			log.Printf("D! Can not parse value: %v for key: %v", field.Value, field.Key)
 			continue
+		case bool:
+			if field.Value == bool(true) {
+				// Store 1 for a "true" value
+				field.Value = 1
```

Comment:
Can you change this to copy to a local?  I'm planning to change around FieldList() to prevent modification this way, it will give more predictable behavior during retries and harder to use the type incorrectly.

Might be easiest to rename `verifyValue` and have it return a value + bool so we don't have to switch twice:
```
func convertValue(v interface{}) (interface{}, bool)
```

Reply:
went back to having it called verifyValue() but I believe I implemented it in a fashion compatible with your request.

Reply:
(I don't know what the protocol is for who should resolve the conversation)